[16.0][Add] sale_order_blanket_order (#3436)
OCA/sale-workflow: sale_order_blanket_order/readme/CONTEXT.md

When a company sells the same products to the same customers on a regular basis, it's a common business practice to create a sale framework that defines the terms and conditions of the sales.

> 

Reviewer: ```suggestion
When a company sells the same products to the same customers on a regular basis, it's a common business practice to create a blanket order that defines the terms and conditions of the sales.

```

If you need a way to define:
* the terms and conditions of the sales,
* the payment terms,
* the delivery terms,

> and also secure the quantities of the products to be delivered, the sale framework module is the right choice.

Reviewer: ```suggestion
and also secure the quantities of the products to be delivered, the sale order blanket order module is the right choice.
```

> This module introduces 2 new kinkds of sale orders:

Reviewer: ```suggestion
This module introduces 2 new kinds of sales orders:
```

> 1. Sale Blanket Order: This is a sale order that defines the terms and conditions of the sales, the payment terms, the delivery terms, and secures the quantities of the products to be delivered. It is used to create sale orders that will be delivered in the future.   

Reviewer: ```suggestion
1. Blanket Order: This is a sales order that defines the terms and conditions of the sales, the price, the payment terms, the delivery terms, and secures the quantities of the products to be delivered.
```

> 2. Call of order: This is a sale order that is created to consume the quantities of the products secured in the sale blanket order.

Reviewer: ```suggestion
2. Call of order: This is a sales order linked to a blanket order that is created to trigger the delivery of quantities of the products secured in the blanket order.
```

> Others modules can be used to provide the same kind of features. For example, the module (sale_blanket_order)[https://pypi.org/project/odoo-addon-sale-blanket-order] also defines the concept of sale blanket order. The main difference between the two modules is that the sale framework module extends the sale order model to add the sale blanket order and the call of order. This allows to keep the benefits of all the extensions made on the sale order model by other modules without having to adapt them to the sale blanket order model (discount, invoicing; inventory process, ...).

Reviewer: ```suggestion
Others modules can be used to provide the same kind of features. For example, the module (sale_blanket_order)[https://pypi.org/project/odoo-addon-sale-blanket-order] also defines the concept of sale blanket order. The main difference between the two modules is that the sale order blanket order module extends the sale order model to add the sale blanket order and the call off order. This allows to keep the benefits of all the extensions made on the sale order model by other modules without having to adapt them to the sale blanket order model (discount, invoicing; inventory process, ...).
```
